sovryn-token-bridge/federator/src/lib/TransactionSender.js

```diff
@@ -46,42 +41,54 @@ module.exports = class TransactionSender {
     }
 
     async getGasLimit(rawTx) {
-        //let estimatedGas = await this.client.eth.estimateGas({ gasPrice: rawTx.gasPrice, value: rawTx.value, to: rawTx.to, data: rawTx.data, from: rawTx.from});
-        let estimatedGas = await this.client.eth.estimateGas({ value: rawTx.value, to: rawTx.to, data: rawTx.data, from: rawTx.from});
+        const estimatedGas = await this.client.eth.estimateGas({
+            value: rawTx.value,
+            to: rawTx.to,
+            data: rawTx.data,
+            from: rawTx.from,
+        });
 
         // Vote: ~70k
         // Vote+execute: A little over 250k
         // First side token deployment: ~3.15M
         const minimum = 300000;
-        estimatedGas = (estimatedGas < minimum) ? minimum : 3500000;
+        const gasLimit = estimatedGas < minimum ? minimum : 3500000;
 
-        return estimatedGas;
+        return gasLimit;
     }
 
     async getChainId() {
-        const chainId =  this.chainId || await this.client.eth.net.getId();
+        const chainId = this.chainId || (await this.client.eth.net.getId());
         return chainId;
     }
 
-    async createRawTransaction(from, to, data, value, privateKey) {
+    getChainName(chainId) {
```

Comment:
I feel like it is not essential since there is no other function used only for none Eth chains ? plus this function might by migrated to utils and return more chain names if ever needed somewhere else in the future ?